clusterloader2/pkg/test/simplecontext.go

```diff
@@ -19,23 +19,26 @@ package test
 import (
 	"k8s.io/perf-tests/clusterloader2/pkg/config"
 	"k8s.io/perf-tests/clusterloader2/pkg/framework"
+	"k8s.io/perf-tests/clusterloader2/pkg/measurement"
 	"k8s.io/perf-tests/clusterloader2/pkg/state"
 	"k8s.io/perf-tests/clusterloader2/pkg/ticker"
 )
 
 type simpleContext struct {
-	framework        *framework.Framework
-	state            *state.NamespacesState
-	templateProvider *config.TemplateProvider
-	tickerFactory    ticker.TickerFactory
+	framework          *framework.Framework
+	state              *state.NamespacesState
+	templateProvider   *config.TemplateProvider
+	tickerFactory      ticker.TickerFactory
```

Comment:
This is oddly the only non-pointer type. Can you change it?

Reply:
This `TickerFactory` is an interface - it doesn't use * cause *type is already cast to this interface.

Reply:
Ok.